### ISSUE TITLE
Vibrance value is set incorrectly

### ISSUE THREAD

**Issue by guihkx** [zee-mzha/vibrantLinux#16]
Hello!

It appears that Vibrant Linux is not properly calculating the initial Vibrance value of my display. I'm using NVIDIA and I'm on commit b2798b4.

Reproduction steps:

1. Start fresh, remove the config file:

    ```
    $ rm ~/.config/vibrantLinux/vibrantLinux.internal
    ```

2. Use `nvidia-settings` to query the current `DigitalVibrance` value (we'll use it later for comparison purposes):

    ```
    $ nvidia-settings -q '[DPY:DVI-D-0]/DigitalVibrance'
    0
    ```

3. Run Vibrant Linux. First notice the Vibrance slider sitting in the wrong position (`1`), when it should be `100`:

    ![Screenshot from 2020-05-31 19-10-29](https://user-images.githubusercontent.com/626206/83363775-6d165200-a372-11ea-9a17-e2f30ec97dd4.png)

4. Close Vibrant Linux, and check the config file (notice that `vibrance` is set to `1`):

    ```
    $ cat ~/.config/vibrantLinux/vibrantLinux.internal
    {
        "configVersion": 1,
        "displays": [
            {
                "name": "DVI-D-0",
                "vibrance": 1
            }
        ],
        "programs": [
        ]
    }

    ```

5. Re-open Vibrant Linux, wait a couple of seconds.
6. Watch your display turn gray. (😆)
7. Finally, query the current `DigitalVibrance` value again, so we can compare with the first one:

    ```
    $ nvidia-settings -tq '[DPY:DVI-D-0]/DigitalVibrance'
    -1013
    ```

**Comment by Scrumplex** [comment]
This has been fixed with commit 43e865de7e6443015874e197d6a0964d6e18fad8.

Stay tuned for the next release

**Comment by guihkx** [comment]
Are you sure? The testing I did was on b2798b4 (two commits after 43e865d)

**Comment by Scrumplex** [comment]
What happens when you delete your config (and let it regenerate)?

**Comment by guihkx** [comment]
Then Vibrant Linux writes the wrong vibrance value to the config file.

The initial `DigitalVibrance` of my display is `0`:

```
$ nvidia-settings -tq '[DPY:DVI-D-0]/DigitalVibrance'
0
``` 

So when I launch Vibrant Linux, it sets the vibrance value on both the slider and config file to `1` (when it should be `100`). 

Speaking of which, I don't quite understand why the range of the vibrance slider is 0 to 400, to be honest. It used to be 0 to 100 IIRC.

**Comment by zee-mzha** [comment]
oh woop, sorry i didn't see your request. This has been fixed in v2.1.2's https://github.com/zee-mzha/vibrantLinux/commit/9f6bd718e9215449105575ce452894d65a1997fd  commit, thank you for reporting this.

**Comment by guihkx** [comment]
No worries, mate!

Thanks for fixing it! 😄
